internal/common/db.go

```diff
@@ -188,12 +188,12 @@ func GetClustersFromDBWhere(db *gorm.DB, eagerLoading EagerLoadingState, include
 	return clusters, nil
 }
 
-func GetHostFromDB(db *gorm.DB, clusterId, hostId string) (*Host, error) {
+func GetHostFromDB(db *gorm.DB, infraEnvId, hostId string) (*Host, error) {
 	var host Host
 
-	err := db.First(&host, "id = ? and cluster_id = ?", hostId, clusterId).Error
+	err := db.First(&host, "id = ? and infra_env_id = ?", hostId, infraEnvId).Error
```

Comment:
All the locations that use this function were not changed - since they are still passing cluster-id.  Why change it ?

Reply:
Starting from this commit we are may be using unbound hosts, without cluster id, so the function must be updated. The whole move from cluster-id to infra-env will be done in MGMT-7321